Production_scripts/Create_costs/costs-care_home.R

```diff
@@ -88,9 +97,7 @@ ch_costs_uplifted <-
 # Join data together  -----------------------------------------------------
 
 # match files - to make sure costs haven't changed radically
-old_costs <- haven::read_sav(
-  get_ch_costs_path(update = latest_update())
-) %>%
+old_costs <- haven::read_sav(get_ch_costs_path(update = latest_update())) %>%
```

Comment:
getting an error reading this file in

`Error: Failed to parse /tmp/Rtmp5rZFxS/file93037b8b0e45: Invalid file, or file has unsupported features.` 

can't see any change between the old way and new? has `get_ch_costs_path` changed?

Reply:
`gte_ch_costs_path()` still `.sav` on main-R so still needs merged